references/video_classification/datasets.py

```diff
@@ -1,14 +1,18 @@
 from typing import Tuple
 
-import torchvision
-from torch import Tensor
+import torch
+from torchvision import datasets
+from torchvision.prototype import features
 
 
-class KineticsWithVideoId(torchvision.datasets.Kinetics):
-    def __getitem__(self, idx: int) -> Tuple[Tensor, Tensor, int]:
+class KineticsWithVideoId(datasets.Kinetics):
+    def __getitem__(self, idx: int) -> Tuple[torch.Tensor, torch.Tensor, features.Label, int]:
```

Comment:
Not sure why we have annotations here, but they were wrong before. Note that type of the first returned value is not `features.Video`, since we normalize before.

Reply:
We don't need the annotations, you are right.